test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1754,8 +1754,8 @@ def test_cgroup_offline_node_vnpernuma(self):
         the cgroup and brought back online once the cgroup is cleaned up.
         vnode_per_numa_node = true
         """
-        with open('/proc/meminfo', 'r') as desc:
-            if 'Hugepagesize' not in desc:
+        meminfo = open(os.path.join(os.sep, 'proc', 'meminfo'), 'r').read()
+        if 'Hugepagesize' not in meminfo:
```

Comment:
I think you still want the with statement.  I don't believe the file will ever be closed.  I think you want the with open() as desc.  Then if 'Hugepagesize' not in desc.read()